frontend/src/components/news-page/News.tsx

```diff
@@ -1,3 +1,43 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 
-export const News: React.FunctionComponent = () => <></>;
+export const News: React.FunctionComponent = () => {
+  const [articlesList, setArticlesList] = useState<any[]>([]);
+
+  useEffect(() => {
+    const url = "https://www.reddit.com/r/whales/top.json?limit=50&t=month";
+
+    const fetchData = async () => {
+      try {
+        const response = await fetch(url);
+        const json: any = await response.json();
+        const articles = json.data.children;
```

Comment:
Please move this code out into the above-mentioned `src/clients/redditApiClient.ts`, into a function that probably looks like
```
export const getTopWhaleVideos = async (): Promise<RedditVideo> => {
  ...
}
```
where `RedditVideo` probably isn't the best name for the interface, but whatever you pick, it gets returned from the method